details.js

```diff
@@ -10,12 +10,34 @@ async function handleFetchFood(){
   // step 1: fetch from server
   // let result = await fetch("https://world.openfoodfacts.org/api/v3/product/078742040370.json");
   // let result = await fetch("https://pokeapi.co/api/v2/item/1");
-  let result = await fetch(`https://pokeapi.co/api/v2/berry/${berryCode}`);
-  console.log(result);
-  // step 2: read data as JSON.
-  let data = await result.json();
-  // proof of life for data
-  console.log(data);
+  let result;
```

Comment:
We need this because the result variable need to be declared outside of the try block.